mas-foundation/src/main/java/com/ca/mas/core/io/ssl/TLSSocketFactory.java

```diff
@@ -68,8 +72,14 @@ public Socket createSocket(InetAddress address, int port, InetAddress localAddre
     }
 
     private Socket enableTLS(Socket socket) {
-        if (socket != null && (socket instanceof SSLSocket)) {
-            ((SSLSocket) socket).setEnabledProtocols(SUPPORTED_TLS);
+        if ((socket instanceof SSLSocket)) {
```

Comment:
Why is the null check removed here ?

Reply:
Because instanceof covers that null check. It is actually redundant. IDE was showing the same.